Aragon Monthly 6th Edition

tao opened the pull request:
# Compile Aragon Monthly

### Closing of a referenced Issue
Closes #172 

### Content category
_Article_ / _Column_ / _News_ / _Classified_

#### Content type:
_Editorial_

### Title
Aragon Monthly 6th Edition

### Description
Contains all the news for Issue 06, and one classifieds article. As far as I understand there are no Articles this month? I addd some suitable images for missing images themselves, and wrote custom excerpt for each article. 

Issue05 was archived and linked in the mkdocs.yml file.

On the font page I've taken the 6 most recent compelling articles, but I'm happy to switch them out if want to feature different articles. 

Let me know if you have any changes or would like to feature any articles this month.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/aragon/aragon-monthly?pullRequest=175) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/aragon/aragon-monthly?pullRequest=175) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/aragon/aragon-monthly?pullRequest=175) it.</sub>

tao commented:
The custom images I used in the articles are:

[Britannica](https://www.britannica.com/topic/New-York-Central-Railroad-Company) (As far as I know there's no copyright issue with using this image from Britannica.)
- **Crypto Commons**

[Unsplashed](https://unsplash.com)
- **What is on-chain cryptocurrency governance? Is it plutocratic?**
- **Towards An International Court of Smart Contract Arbitration**
- **Ethereum’s Forgotten Treasure: DAOs**

The other images are from the articles or links themselves.

Smokyish commented:
@tao Thank you for the submission! There are 3 Articles under Opinion pieces that have been merged into `master` for Issue 06.

Will start a review of the PR and if you can in the mean time add the articles, that's great, if not, that's not a blocking issue.

tao commented:
Right, sorry about that - I removed them because I believed they were from issue05. I've added them back in and done some editing, and added them to the front-page `index.md` and `mkdocs.yml` file too.

Some issues with the opinion pieces that I'm not sure how to deal with are:

**The importance of Governance:** 
The image mentioned in the markdown is missing, I'm not sure where to find that... its looking for (images/dao.jpg)

**Ethereum’s response to EIP0 Results: Governance & Community**
The first heading has code comments: "**// The first...**" That's how it looks in the medium article but we may want to remove it so its just: "**The first...**"

Are there any other issues remaining?

Smokyish commented on docs/news/aragon.md:
```diff
@@ -1,53 +1,20 @@
-**First Aragon app launched on Ethereum Mainnet for community sentiment signaling**. Today we are releasing an entirely new step for community members to signal their sentiment on the project and igniting the next level of participation in the project for all ANT holders. Survey app is the fourth app developed by our team in addition to the Token Manager, Voting & Finance apps seen previously in  ...
-[Read More](https://blog.aragon.one/aragon-launches-survey-app-on-mainnet-ed5eefeb66f5) | [Read More](../news) | [Submit News!](../guides/guide_for_submitting_news.md) |
+[_Aragon_](aragon.md) | | |
+[<img src="https://cdn-images-1.medium.com/max/1000/1*-hd2CC4ZPZ6N5wmhH7YTZA.jpeg">](https://blog.aragon.one/aragon-core-v0-5-1-post-mortem-part-1-85b0897b9157) | | |
```
Last, empty, columns could use the `../../images/monthly_no_image.png` image

Smokyish commented on docs/news/daos.md:
```diff
@@ -1,21 +1,11 @@
-|Author [Chelsea Rustrum](https://hackernoon.com/@shareablelife) on Apr 4|Author [Darren Heitner](https://www.inc.com/author/darren-heitner) on Apr 27||
-|I’d experienced the joy of connecting without money — sharing meals and homes with complete strangers on Couchsurfing.com, gifting access to my car while I traveled, and experimenting with random acts of generosity. These experiences were so transformational that I volunteered to help shoot a documentary, One Couch at a Time, which highlighted the relational aspects of the burgeoning sharing eco ...
-|[Read More](https://hackernoon.com/should-couchsurfing-be-a-dao-6507646e34ef)|[Read More](https://www.inc.com/darren-heitner/never-heard-of-a-dao-heres-why-its-going-to-become-hottest-new-type-of-corporation.html)||
+[**How Daox Works — Part 1**](https://medium.com/daox/how-daox-works-part-1-a1d2a456cbe7) | [**Ethereum’s Forgotten Treasure: DAOs**](https://cryptoslate.com/ethereums-forgotten-treasure-daos/) |
```
Missing the third, empty, column.

Smokyish commented on docs/news/governance.md:
```diff
@@ -1,27 +1,35 @@
+[**Voting and Governance in Security Tokens**](https://medium.com/coinmonks/voting-and-governance-in-security-tokens-1e3d041dabb8) | [**Crypto Commons — from tragedy to wealth of the commons**](https://blog.usejournal.com/crypto-commons-da602fb98138) | |
+:-----------:|:-----------:|:-----------:|
+[_Governance_](governance.md) | [_Governance_](governance.md) | |
+[<img src="https://cdn-images-1.medium.com/max/800/1*F7oHt_loAWz4EACe_SqRgQ.jpeg">](https://medium.com/coinmonks/voting-and-governance-in-security-tokens-1e3d041dabb8) | [<img src="https://cdn.britannica.com/668x448/09/134309-004-ABF9FE28.jpg">](https://blog.usejournal.com/crypto-commons-da602fb98138) | |
```
Last, empty, column could use the `../../images/monthly_no_image.png` image and the `Submit News!` link.

Smokyish commented on docs/news/index.md:
```diff
@@ -1,69 +1,51 @@
-| **Launch of a new edition of the Aragon Network whitepaper and our research forum**. The initial version of the Aragon Network whitepaper was published over a year ago. Since then we have made tremendous progress in making Decentralized Autonomous Organizations a reality. |As a human race we started taking governance seriously during the Renaissance period. We started to use our rationality to  ...
-| [Read More](https://blog.aragon.one/aragon-network-on-a-path-towards-a-digital-jurisdiction-752ccf4b9d5c) |[Read More](https://medium.com/betoken/we-need-to-formalize-governance-research-using-daos-e9393da427f3)| [Read More](https://blog.aragon.one/aragon-launches-survey-app-on-mainnet-ed5eefeb66f5) |
+[_Governance_](governance.md) | [_Governance_](governance.md) | |
+[<img src="https://cdn-images-1.medium.com/max/800/1*F7oHt_loAWz4EACe_SqRgQ.jpeg">](https://medium.com/coinmonks/voting-and-governance-in-security-tokens-1e3d041dabb8) | [<img src="https://cdn.britannica.com/668x448/09/134309-004-ABF9FE28.jpg">](https://blog.usejournal.com/crypto-commons-da602fb98138) | |
```
Last, empty, column could use the `../../images/monthly_no_image.png` image and the `Submit News!` link.

Smokyish commented:
@tao Great work, will merge this as is, please see our [How to claim an Aragon bounty using Status Open Bounty](https://wiki.aragon.one/tutorials/Status_Open_Bounty_Guide/) for claiming the bounty, payouts are done weekly so you will receive the payout within a week.
